common/views/components/Body/Body.js

```diff
@@ -172,14 +174,49 @@ const Body = ({ body, isDropCapped, pageId }: Props) => {
             )}
 
             {slice.type === 'collectionVenue' && (
-              <VenueHours
-                venue={parseVenueTimesToOpeningHours(slice.value)}
-                isInList={
-                  pageId === 'openingTimes' || pageId === 'WwQHTSAAANBfDYXU'
-                }
-              />
+              <>
+                {slice.display && (
```

Comment:
Trying to think about how we remove these conditionals on page ID and turn them into abstract concepts.
A few things I can think of is using the labels to work with display logic - which is already the case.

Maybe the slice could have a `boolean` saying `showClosingHours` instead of a label, and then the label could be used to decide between the wide or not wide grid negating the need for this very ❄️ code?